sass/components/_announcement-bar.scss

```diff
@@ -1,6 +1,38 @@
 @charset 'UTF-8';
 
 .announcement-bar {
+    transition: 0.2s ease-in-out;
+    &.tall {
+        height: 40px;
+        max-height: 40px;
+        min-height: 40px;
+        padding: 0;
+        margin: 0;
+        > span {
+            font-size: 14px;
+            line-height: 20px;
+            font-weight: 600;
```

Comment:
This is causing the icon to be bold. The font-weight should probably be set on the text only